manager/allocator/cnmallocator/networkallocator.go

```diff
@@ -49,7 +49,7 @@ type cnmNetworkAllocator struct {
 
 	// Allocator state to indicate if allocation has been
 	// successfully completed for this node.
-	nodes map[string]struct{}
+	nodes map[string]map[string]struct{}
```

Comment:
It would be helpful for the comment to explain what the keyspace of the inner map is.